doc/user/methods/random_forest.md

```diff
@@ -367,6 +373,41 @@ std::cout << "Probabilities of each class: " << probabilities.t();
 
 ---
 
+Train a `RandomForest` with the `SequentialBootstrap` strategy.
```

Comment:
Either we should locate this after the section of documentation that introduces the `SequentialBootstrap`, or we should link ahead to the documentation.